config/sevntu_suppressions.xml

```diff
@@ -40,4 +40,7 @@
   <!-- No need for constructor with cause -->
   <suppress checks="CauseParameterInException" files=".*MetadataGenerationException\.java"/>
 
+  <!-- Migrating to Truth -->
+  <suppress checks="RequireFailForTryCatchInJunit" files="MutableExceptionCheckTest.java"/>
```

Comment:
This suppression is not needed.
The bug will be fixed in sevntu-checkstyle/sevntu.checkstyle#847

Reply:
@shashwatj07 please subscribe to issue that @pbludov provided, to get notification when issue is closed

Reply:
while bug is not fixed and released , we need to use suppression.

Reply:
@pbludov , please reconsider blocking such PRs, yes now it requires extra work, but it is not blocked and all suppressions could be removed ones sevntu issue is resolved.

Reply:
sure, we can merge this first.